src/main/java/org/jboss/modules/JarModuleFinder.java

```diff
@@ -171,4 +173,13 @@ public ModuleSpec findModule(final String name, final ModuleLoader delegateLoade
             return moduleSpec;
         }
     }
+
+    private static String toPathString(String moduleName) {
+        return moduleName.replace('.', '/') + '/' + moduleName + '/';
```

Comment:
Method body should be: 

return moduleName.replace('.', '/');

without ending slash, because later this path is used in combination with "/module.xml" string.